build/components/layout/iframe-pages/layout_large.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
@maciekbb can we exclude this path: `iframe-pages`? it looks like this is used only for building demo page